src/Compilers/CSharp/Test/Semantic/Semantics/RecordStructTests.cs

```diff
@@ -4903,25 +4911,21 @@ public void ToString_TopLevelRecord_OneField_ValueType()
 
             v.VerifyIL("C1." + WellKnownMemberNames.PrintMembersMethodName, @"
 {
-  // Code size       50 (0x32)
+  // Code size       38 (0x26)
```

Comment:
I'd say the reduction in IL size is probably the biggest benefit. I think that's nice. Thanks